Tutorials/P5JS/p5.js/18/18.01_p5.js_Intro_to_webGL/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>InstanceMode</title>
+    
+    <script src="libraries/p5.js" type="text/javascript"></script>
+    <script src="libraries/p5.dom.js" type="text/javascript"></script>
+
+    <script src="sketch.js" type="text/javascript"></script>
+
+    <style> body {padding: 0; margin: 0;} canvas {vertical-align: top;} </style>
```

Comment:
This is so silly, but I generally prefer to leave this out (unless the canvas is specifically filling the page.)